### PR TITLE
DM-3018: Made each practice action navigable, more ARIA-friendly, and…

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3018

## Description - what does this code do?
Makes the practice actions (bookmark, share, and print) navigable via keyboard and improves ARIA support.

## Testing done - how did you test it/steps on how can another person can test it 
1. Visit any practice page and make sure you can tab through each link.
2. **BONUS:** Use a screen reader and ensure the labels are being read correctly (you can refer to the gif below if need be).

## Acceptance criteria
- Make sure the screen reader can navigate to the subscribe/print/share buttons on the practice page

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/practices/show/introduction/_introduction.html.erb [hunk 1]
```diff
@@ -24,33 +24,28 @@
...
-      </div>
+      <% end %>
+      <%# share %>
+      <a target="_blank" aria-label="Share <%= @practice.name %> via email" href="mailto:?subject=<%= email_practice_subject(@practice) %>&body=<%= email_practice_body(@practice) %>"
+         class="dm-icon-link text-no-underline display-inline-block text-primary-vivid font-sans-xs margin-bottom-2 margin-right-5">
+        <i class="far fa-share-square margin-right-05"></i>
+        <span>Share</span>
+      </a>
+      <%# print %>
+      <a class="dm-icon-link text-no-underline display-inline-block font-sans-xs text-primary-vivid margin-bottom-2" onclick="window.print(); return false;"
```
Comment (emyl3) [inline app/views/practices/show/introduction/_introduction.html.erb:44]: the other two links work when I tab to it and press "Enter" but the print one doesn't. We should add a separate event listener to ensure the print button works on enter.

**Review by emyl3 — CHANGES_REQUESTED** [review]
one thing unless you think we need a separate story for this :) Good work on this Josh! The voiceover worked perfectly on my machine :)

File: app/views/practices/show/introduction/_introduction.html.erb (hunk 1 above)
Comment (JoshingYou1) [reply app/views/practices/show/introduction/_introduction.html.erb:44]: I think I can add that to this story. I'll get back to you. Thanks Elisa!

**Review by emyl3 — APPROVED** [review]
very nice!!

**Review by bradjohnson92008 — APPROVED** [review]
Worked for me.. great work Josh!

**Pull request merged**